chainlink-testing-framework/run-tests/action.yml

```diff
@@ -184,12 +227,39 @@ runs:
       shell: bash
       run: go install github.com/smartcontractkit/chainlink-testing-framework/tools/gotestloghelper@v1.1.1
 
+    - name: Set default test secrets
+      shell: bash
+      run: |
+        echo "TEST_CONFIG_CHAINLINK_IMAGE=${{ inputs.DEFAULT_TEST_CONFIG_CHAINLINK_IMAGE }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_LOKI_TENANT_ID=${{ inputs.DEFAULT_TEST_CONFIG_LOKI_TENANT_ID }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_LOKI_ENDPOINT=${{ inputs.DEFAULT_TEST_CONFIG_LOKI_ENDPOINT }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_LOKI_BASIC_AUTH=${{ inputs.DEFAULT_TEST_CONFIG_LOKI_BASIC_AUTH }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_LOKI_BEARER_TOKEN=${{ inputs.DEFAULT_TEST_CONFIG_LOKI_BEARER_TOKEN }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_GRAFANA_BASE_URL=${{ inputs.DEFAULT_TEST_CONFIG_GRAFANA_BASE_URL }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_GRAFANA_DASHBOARD_URL=${{ inputs.DEFAULT_TEST_CONFIG_GRAFANA_DASHBOARD_URL }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_GRAFANA_BEARER_TOKEN=${{ inputs.DEFAULT_TEST_CONFIG_GRAFANA_BEARER_TOKEN }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_PYROSCOPE_SERVER_URL=${{ inputs.DEFAULT_TEST_CONFIG_PYROSCOPE_SERVER_URL }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_PYROSCOPE_KEY=${{ inputs.DEFAULT_TEST_CONFIG_PYROSCOPE_KEY }}" >> $GITHUB_ENV    
+        echo "TEST_CONFIG_PYROSCOPE_ENVIRONMENT=${{ inputs.DEFAULT_TEST_CONFIG_PYROSCOPE_ENVIRONMENT }}" >> $GITHUB_ENV
+        echo "TEST_CONFIG_PYROSCOPE_ENABLED=${{ inputs.DEFAULT_TEST_CONFIG_PYROSCOPE_ENABLED }}" >> $GITHUB_ENV
```

Comment:
wondering how safe that is to use `echo`, I think this syntax was a bit better, because it wasn't printing anything at all:
```
TEST_CONFIG_PYROSCOPE_ENABLED =$(jq -r '.inputs. DEFAULT_TEST_CONFIG_PYROSCOPE_ENABLED' $GITHUB_EVENT_PATH)
```

wdyt?